x-pack/plugins/security_solution/public/common/components/navigation/breadcrumbs/index.ts

```diff
@@ -136,7 +140,11 @@ const getTrailingBreadcrumbsForRoutes = (
   return [];
 };
 
-const emptyLastBreadcrumbUrl = (breadcrumbs: ChromeBreadcrumb[]) => {
+const emptyLastBreadcrumbUrl = (breadcrumbs: ChromeBreadcrumb[], isExistingRule?: boolean) => {
+  if (!isExistingRule && isBoolean(isExistingRule)) {
+    breadcrumbs.push({ text: i18n.DELETED_RULE, href: '/' });
+  }
```

Comment:
This `emptyLastBreadcrumbUrl` function, as well as other breadcrumbs-related code (including the `TabNavigationWithBreadcrumbs` component), being generic and common to the whole security solution app, should not know about whether a rule exists or is deleted, or rules in general.

I'd suggest to revert changes to the breadcrumbs implementation and let this implementation continue to operate over abstractions, e.g. the current abstraction of tabs (see `ObjectWithNavTabs`).